wlct/tournaments.py

```diff
@@ -2501,6 +2506,7 @@ class TournamentGame(models.Model):
     gameid = models.CharField(max_length=255, default="Invalid game id")
     tournament = models.ForeignKey('Tournament', on_delete=models.CASCADE)
     teams = models.CharField(max_length=255, null=True, db_index=True)
+    players = models.CharField(max_length=255, null=True, db_index=True)
```

Comment:
I don't see this ever being used in a query. If not let's remove it, it adds a new index which is just database overhead that's not necessary until we are querying directly against this.